### PR TITLE
merge: develop to main

### PR DESCRIPTION
**Pull request by sunlight0602**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,4 +17,6 @@ pip uninstall numpy
 pip install numpy==1.23
 python main.py
```
Comment (TWcamel) [inline README.md:19]: add pip install pyobjc to doc.

**Review by TWcamel — COMMENTED** [review]
see comment

**Review by TWcamel — CHANGES_REQUESTED** [review]
see comments

**Comment by sunlight0602** [comment]
@TWcamel add pip install pyobjc to doc

File: README.md (hunk 1 above)
Comment (TWcamel) [reply README.md:19]: well done

**Review by TWcamel — APPROVED** [review]
well done

**Pull request merged**